src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -50,10 +62,14 @@ const CrossPodWriteForm = () => {
     clearDescription();
     dispatch({ type: 'CLEAR_FILE' });
     dispatch({ type: 'CLEAR_PROCESSING' });
+    // setDocType('');
+    // setExpireDate(null);
+    // setSelectedUser('');
```

Comment:
This is being phased out in #246

Reply:
Once we know everything is working we can remove the comment in Line 60 and Line 66 since they're no longer in use.